app/src/main/java/com/duckduckgo/app/statistics/store/PixelDao.kt

```diff
@@ -16,14 +16,18 @@
 
 package com.duckduckgo.app.statistics.store
 
-import androidx.room.Dao
-import androidx.room.Query
-import com.duckduckgo.app.cta.model.DismissedCta
+import androidx.room.*
 import com.duckduckgo.app.statistics.model.PixelEntity
 import kotlinx.coroutines.flow.Flow
 
 @Dao
 interface PixelDao {
     @Query("select * from pixel_store")
-    fun pixels(): Flow<PixelEntity>
+    fun pixels(): Flow<List<PixelEntity>>
```

Comment:
Along with previous comment, what do you think about making it return an Observable so that we avoid crossing Flow<>RxJava boundaries?